doc/api/addons.md

```diff
@@ -107,6 +107,14 @@ When building addons with `node-gyp`, using the macro `NODE_GYP_MODULE_NAME` as
 the first parameter of `NODE_MODULE()` will ensure that the name of the final
 binary will be passed to `NODE_MODULE()`.
 
+If you define addon with `NODE_MODULE`, The usage is as follows.
+
+1. If you load the addon in main thread first, you can not load the addon
+   in worker.
+2. If you load the addon in worker first, you can not load the addon in
+   main thread or other workers until the worker which have loaded the addon
+   exits.
```

Comment:
I think this could be made a bit more succinct, e.g. “Addons defined with `NODE_MODULE()` cannot be loaded in multiple contexts or multiple threads at the same time.”?

I wouldn’t make the text here too verbose since the next section already touches on this quite a bit.